Makefile

```diff
@@ -92,6 +92,9 @@ lint: check-venv
 docs: check-venv
 	cd docs && $(MAKE) html
 
+serve-docs: docs
+	@cd docs/_build/html && python3 -m http.server
```

Comment:
Since the `docs/Makefile` already has a `make serve`, it might make sense to do `        cd docs && $(MAKE) serve` instead.

Reply:
Good catch. Let's use that.